source/installation/with/prebuilt.html.md

```diff
@@ -2,5 +2,43 @@
 title: Installing pre-built binaries
 ---
 
-This installation method is not yet available. We will provide pre-built binaries
-that you can intall and run in a few seconds before the first official release.
\ No newline at end of file
+This installation method is only available for 64 bits Linux machines. Pre-built binairies
+for other platforms may come in the future.
+
+You will need to have `wget` installed for this method to work.
+
+To download and install the latest release, first define the following variables in your shell:
+
+- `PLUME_VERSION`: the Plume version to install. The latest is `0.3.0-alpha-2`.
+- `PLUME_DB`: the database backend to use. Either `postgres` or `sqlite`.
+
+Then run these commands:
+
+```bash
+# Create a directory to store executables (this is the standard place for Rust binaries)
+mkdir -p ~/.cargo/bin
+# Create a directory to store static files and database config
+mkdir Plume
+cd Plume
+
+# Download the files
+wget -O plume.tar.gz TODO
+# Extract them
+tar -xvzf plume.tar.gz
```

Comment:
`tar xf plume.tar.gz` should be enough, really.

Reply:
(to be honest I copied a command from stackoverflow, because I can't remember how to use `tar` :sweat_smile:)